apps/dashboard/src/components/workflow-editor/create-workflow-form.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormInput,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/primitives/form/form';
+import { Separator } from '@/components/primitives/separator';
+import { TagInput } from '@/components/primitives/tag-input';
+import { Textarea } from '@/components/primitives/textarea';
+import { useTags } from '@/hooks/use-tags';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { type CreateWorkflowDto, slugify } from '@novu/shared';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+import { MAX_DESCRIPTION_LENGTH, MAX_TAG_ELEMENTS, workflowSchema } from './schema';
+
+interface CreateWorkflowFormProps {
+  onSubmit: (values: z.infer<typeof workflowSchema>) => void;
+  template?: CreateWorkflowDto;
+}
+
+export function CreateWorkflowForm({ onSubmit, template }: CreateWorkflowFormProps) {
+  const form = useForm<z.infer<typeof workflowSchema>>({
+    resolver: zodResolver(workflowSchema),
+    defaultValues: {
+      description: template?.description ?? '',
+      workflowId: template?.workflowId ?? '',
+      name: template?.name ?? '',
+      tags: template?.tags ?? [],
+    },
+  });
+
+  const { tags } = useTags();
+  const tagSuggestions = tags.map((tag) => tag.name);
+
+  return (
+    <Form {...form}>
+      <form
+        id="create-workflow"
+        autoComplete="off"
+        noValidate
+        onSubmit={form.handleSubmit(onSubmit)}
+        className="flex flex-col gap-4"
+      >
+        <FormField
+          control={form.control}
+          name="name"
+          render={({ field }) => (
+            <FormItem>
+              <FormLabel required>Name</FormLabel>
+              <FormControl>
+                <FormInput
+                  {...field}
+                  autoFocus
+                  onChange={(e) => {
+                    field.onChange(e);
+                    form.setValue('workflowId', slugify(e.target.value));
+                  }}
+                />
+              </FormControl>
```

Comment:
please adjust the sizes of all inputs as it hearts to see that they have different font sizes, for example description is the smallest
![Screenshot 2025-01-14 at 17 31 24](https://github.com/user-attachments/assets/23b8180e-d8fc-4813-86c4-c47417a02353)